reference/orders.v2.oas2.yml

```diff
@@ -1534,10 +1526,10 @@ components:
                   street_2: ''
                   city: Austin
                   state: Texas
-                  zip: '78108'
+                  zip: 78108
                   country: United States
                   country_iso2: US
-                  phone: '1234567890'
+                  phone: 1234567890
```

Comment:
Making a `GET` request to `/orders/{order_id}` returns both phone as a string when I test it.